### ISSUE TITLE
[Bug]: Electron 12.2.2 does not work in Linux with multiple GPU when main GPU is not first.

### ISSUE THREAD

**Issue by danilw** [electron/electron#31458]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.

### Electron Version

12.2.2

### What operating system are you using?

Other Linux

### Operating System Version

Opensuse Tumbleweed

### What arch are you using?

x64

### Last Known Working Electron version

13.5.2

### Expected Behavior

Works.

### Actual Behavior

Electron 12.2.2 does not work in Linux with multiple GPU when main GPU is not first.
Example - Integrated GPU with Discrete GPU and Monitor connected to Discrete GPU.

```
[10731:1018/122033.336041:FATAL:gpu_data_manager_impl_private.cc(415)] GPU process isn't usable. Goodbye.
```
Electron 13.5.2 and every new version works.

**Comment by ckerr** [comment]
Hey @danilw, I'm a little confused by the versions you mentioned -- You say it's working in 13.5.2 but not in 12.2.2. Is that correct, or are those versions reversed?

Also, does the problem still persist in the [latest stable release](https://www.electronjs.org/releases/stable)?

**Comment by danilw** [comment]
> Hey @danilw, I'm a little confused by the versions you mentioned -- You say it's working in 13.5.2 but not in 12.2.2. Is that correct, or are those versions reversed?

yes this is correct, my point is - **99.9% of popular mainstream production software that based on Electron stuck on 12.2.2** and who know when they will update to new version (maybe 1 year or 10 years latter or never), **example** - **Discord Skype, and about 3 other "software launchers"(made by very large corporations)** I use every of it dont work on Linux on PC with 2+ GPUs (because they use native 12.2.2 Electron). **And maybe they will update their Electron integration if official 12.x.x will be updated...**

I do not know Electron version order, and **I thought that 12.2.2 listed as "stable" and still supported, this why I created this bugreport.**

**If this is wrong just delete it**, I dont care about these large corporation software who too lazy to check if their software actually works.
